src/playwright-runner.js

```diff
@@ -127,76 +127,59 @@ async function run (nodeBin, runCfgPath, suiteName) {
   const runCfg = await loadRunConfig(runCfgPath);
   runCfg.path = runCfgPath;
   const cwd = process.cwd();
-  let defaultArgs = {
-    param: {
-      headful: process.env.SAUCE_VM ? true : false,
```

Comment:
Aren't these default values required?

Reply:
This have been fixed here: https://github.com/saucelabs/sauce-playwright-runner/pull/99/files#diff-666450f3c804e34e85768192da8dae7fd947b4b14db90b48f12cea82711fcf22R141